encord/user_client.py

```diff
@@ -628,20 +629,34 @@ def get_ontologies(
         return retval
 
     def create_ontology(
-        self, title: str, description: str = "", structure: Optional[OntologyStructure] = None
+        self,
+        title: str,
+        description: str = "",
+        structure: Optional[OntologyStructure] = None,
+        skeleton_templates: Optional[List[SkeletonTemplate]] = None,
     ) -> Ontology:
+        if structure and skeleton_templates:
```

Comment:
OntologyStructure generates a feature_node_hash for the skeleton template. If I add a skeleton object with name "Skeleton", then it is given a FNE which the current logic then uses that for a feature_node_hash for the skeleton_template.